src/framework/theme/components/layout/_layout.component.theme.scss

```diff
@@ -177,6 +177,7 @@
   }
 
   nb-layout-header {
+    background-color: nb-theme(header-background-color);
```

Comment:
do we need the part below then?

```
nav {
      background: nb-theme(header-background-color);
```

Reply:
No, removed in 2fbde0846c84fae87ab49e3743a23d144eb4aef2